roles/network/templates/network/iiab-hotspot-on

```diff
@@ -1,23 +1,21 @@
 #!/bin/bash
-cp -f /etc/hostapd/hostapd.conf.iiab /etc/hostapd/hostapd.conf
-sed -i -e "s/^#denyinterfaces/denyinterfaces/" /etc/dhcpcd.conf
+#cp -f /etc/hostapd/hostapd.conf.iiab /etc/hostapd/hostapd.conf
+#sed -i -e "s/^#denyinterfaces/denyinterfaces/" /etc/dhcpcd.conf
```

Comment:
would need to reference wifi_up_down conditional but in bash as wlan0 would need to removed from /etc/dhcpcd.conf to allow wlan0 to function as a dhcp client and ap0 must be present for hostapd to work.